path_sequence_planner/include/path_sequence_planner/simple_path_sequence_planner.h

```diff
@@ -27,11 +27,13 @@ namespace path_sequence_planner
 class SimplePathSequencePlanner : public PathSequencePlanner
 {
 public:
+  using PathSequencePlanner::PathSequencePlanner;
```

Comment:
Is this line specifying that we are using the constructor from the parent class?

Reply:
Yes; not sure if it's technically required, but I like being explicit about it

Reply:
From [cppreference](https://en.cppreference.com/w/cpp/language/using_declaration), it looks like what this does is add all constructors from the base class for overload resolution when constructing the subclass.  Since this class does not define any constructors, and the parent class only defines the default, I'm not sure this is doing anything other than confusing the next poor fellow to stumble across it.